spec/features/produce_items_spec.rb

```diff
@@ -30,7 +30,7 @@
   it 'expects link to navigate to show and back to index' do
     visit(produce_items_path)
     find('table tr td a.link_action').click
-    find('.link_action').click
+    all('.link_action').last.click
```

Comment:
Should we have a specific class or id for this link? Relying on fetching the last of its class sounds fragile to me.

Reply:
Agreed. Added a back class to all back links